src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Controllers/ReleaseFileControllerTests.cs

```diff
@@ -0,0 +1,120 @@
+        private readonly Mock<IReleaseFileService> _releaseFileService = new(MockBehavior.Strict);
+
+        private readonly Release _release = new()
+        {
+            Publication = new Publication()
+        };
+
+        public ReleaseFileControllerTests(TestApplicationFactory<TestStartup> testApp)
+        {
+            _testApp = testApp.WithWebHostBuilder(
+                builder =>
+                {
+                    builder.AddTestData<ContentDbContext>(
+                        context =>
+                        {
+                            context.Releases.Add(_release);
+                            context.SaveChanges();
+                        }
+                    );
+
+                    builder.ConfigureServices(
+                        services => { services.AddTransient(_ => _releaseFileService.Object); }
+                    );
+                }
+            );
+        }
+
+        public void Dispose()
+        {
+            _releaseFileService.Reset();
+        }
+
+        [Fact]
+        public async Task Stream()
+        {
+            var client = _testApp.CreateClient();
+
+            await using var stream = "Test file".ToStream();
+            var fileId = Guid.NewGuid();
+
+            _releaseFileService
+                .Setup(s => s.StreamFile(_release.Id, fileId))
+                .ReturnsAsync(
+                    new FileStreamResult(stream, MediaTypeNames.Application.Pdf)
+                    {
+                        FileDownloadName = "test-file.pdf",
+                    }
+                );
+
+            var response = await client
+                .GetAsync($"/api/releases/{_release.Id}/files/{fileId}");
+
+            MockUtils.VerifyAllMocks(_releaseFileService);
+
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            Assert.Equal("Test file", await response.Content.ReadAsStringAsync());
+        }
+
+        [Fact]
+        public async Task StreamFilesToZip()
+        {
+            var client = _testApp.CreateClient();
+
+            var fileId1 = Guid.NewGuid();
+            var fileId2 = Guid.NewGuid();
+
+            _releaseFileService
+                .Setup(
+                    s => s.ZipFilesToStream(
+                        _release.Id,
+                        It.Is<IEnumerable<Guid>>(
+                            ids => ids.SequenceEqual(ListOf(fileId1, fileId2))),
+                        It.IsAny<Stream>(),
+                        It.IsAny<CancellationToken?>()
+                    )
+                )
+                .ReturnsAsync(Unit.Instance)
+                .Callback<Guid, IEnumerable<Guid>, Stream, CancellationToken?>(
+                    (_, _, stream, _) => { stream.WriteText("Test zip"); }
+                );
+
+            var response = await client
+                .GetAsync($"/api/releases/{_release.Id}/files?fileIds={fileId1},{fileId2}");
+
+            MockUtils.VerifyAllMocks(_releaseFileService);
```

Comment:
Personally, I lean slightly towards not using the static imports, mainly because it's just a little trouble to actually import them. Rider can't infer what static import you want, so you always end up needing to write `MockUtils`. After that, there's the additional step of having to do the refactoring to the static import. 

It's not a big task, but it's something I rarely bother with given how minor it is. In this situation, I personally don't think the statics add anything, other than pure aesthetics. If we were going to mandate that we always use static imports, we'd have to do it for all the `Assert` methods, which I'm sure no one's going to sign up to do :wink:

Reply:
Hehe, we definitely wouldn't want to mandate using static imports everywhere like with the `Assert` methods! I suppose it's a matter of personal preference like you say.